Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -71,6 +71,13 @@ protected override void Dispose(bool disposing)
 
 			if (disposing)
 			{
+				if (Control != null)
+				{
+					Control.SetOnClickListener(null);
+					Control.RemoveOnAttachStateChangeListener(this);
+					Control.Tag = null;
+					_textColorSwitcher = null;
+				}
```

Comment:
Not here. The base class `ViewRenderer<TView, TNativeView>` still has some cleanup to do on `Control`, then it gets set to `null`.